src/test/java/be/seeseemelk/mockbukkit/entity/PlayerMockTest.java

```diff
@@ -1573,6 +1573,18 @@ void testPlayerPlayEffect_IncorrectData()
 		});
 	}
 
+	@Test
+	void testPlayerSwingMainHand()
+	{
+		assertDoesNotThrow(() -> player.swingMainHand());
+	}
+
+	@Test
+	void testPlayerSwingOffHand()
+	{
+		assertDoesNotThrow(() -> player.swingOffHand());
+	}
+
```

Comment:
Since this functionality is now handled by LivingEntityMock, it should be moved to that test class.